content/en/docs/bmf/getting_started_yourself/install/_index.md

```diff
@@ -391,13 +391,20 @@ To compile libncurses on macOS, you can follow these steps:
 
      d. BMF depends on libbfd, so you need to install binutils
      ```
-     brew install binutils
+     wget https://ftp.gnu.org/gnu/binutils/binutils-2.43.1.tar.bz2
+     tar xvf binutils-2.43.1.tar.bz2
+     cd binutils
+     ./configure --prefix=/usr/local/opt/binutils --enable-install-libiberty
+     sudo make -j8 altinstall
```

Comment:
altinstall is correct, unknow target in my mac

Reply:
sorry, could you try "sudo make -j8 install" ?

Reply:
> sorry, could you try "sudo make -j8 install" ?

it work fine without install, good job!

Reply:
> > sorry, could you try "sudo make -j8 install" ?
> 
> it work fine without install, good job!

If you want to follow the install guide, please install the binutils to /usr/local/opt/binutils.
Because BMF's submodule backward will use it.